backend/src/controllers/job.controller.ts

```
@@ -38,29 +39,41 @@
+  @Security("jwt", ["user"])
   @Tags('Jobs')
   @Get('{queueName}')
   public async getJobs(
-
+    @Request() request: express.Request,
     @Path() queueName: 'jobs',
     @Query() jobId?: string,
     @Query() jobsType?: JobType,
   ): Promise<any> {
+    const scopes = (request as any).user && (request as any).user.scopes
+    const user = (request as any).user && (request as any).user.user
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)